pipelines/cellpainting/cpd_analysis_pipeline.wdl

```diff
@@ -64,6 +68,8 @@ workflow cpd_analysis_pipeline {
     }
   }
 
-
+  output {
+    String segmentation_directory = output_directory
```

Comment:
Does the name `segmentation_directory` always make sense, no matter what analysis cppipe a user writes?

Reply:
Yeah I guess that sounds good

Reply:
Changed in lasted commit